Core/Utilities/Glue2.py

```diff
@@ -134,9 +134,10 @@ def __getGlue2ShareInfo(host, shareInfoLists):
       ceInfo['Queues'] = {}
       queueInfo = {}
       queueInfo['GlueCEStateStatus'] = shareInfoDict['GLUE2ComputingShareServingState']
-      queueInfo['GlueCEPolicyMaxCPUTime'] = str(int(shareInfoDict.get('GLUE2ComputingShareMaxCPUTime', 86400)) / 60)
-      queueInfo['GlueCEPolicyMaxWallClockTime'] = str(int(shareInfoDict
-                                                          .get('GLUE2ComputingShareMaxWallTime', 86400)) / 60)
+      queueInfo['GlueCEPolicyMaxCPUTime'] = str(int(int(
```

Comment:
This is just adding `int(` to an existing `int(`?

Reply:
The output of the `shareInfoDict.get` could be a string, which needs to be casted, no magic cast to default value.

Reply:
There's 
```
int(shareInfoDict.get('GLUE2ComputingShareMaxWallTime', 86400))
```
and then that value is divided by 60: the new `int` cast is for that division.

Reply:
Ah I see it now, sorry failing to pair brackets correctly...